.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: Build APK
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build APK
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Setup Java
+      uses: actions/setup-java@v3
+      with:
+        distribution: temurin
+        java-version: 18
+
+    - name: Build APK
+      run: bash gradlew build
```

Comment:
Shouldn't there be `./gradlew`?

Reply:
I'm not sure if there is execute permission on `gradlew`, so I just run it with shell. I've changed it to `sh`, because `sh` in shebang.